actions/create_xcconfig.rb

```diff
@@ -1,20 +1,20 @@
 module Fastlane
   module Actions
     class CreateXcconfigAction < Action
-      def self.run(params)
+      def self.run(params) # rubocop:disable Metrics/AbcSize
         filename = params[:filename]
         project = Xcodeproj::Project.open(ENV['FL_PROJECT_SIGNING_PROJECT_PATH'])
         lines = []
         project.targets.each do |target|
-          profile = self.get_profile_for_target(target)
-          if !profile.nil?
+          profile = get_profile_for_target(target)
+          if !profile.nil? # rubocop:disable Style/NegatedIf, Style/IfUnlessModifier
```

Comment:
What's the issue with this now being `lines.push("#{target.name}_PROFILE_SPECIFIER=#{profile}") unless profile.nil?`?

Reply:
Nothing! I just explicitly disabled the ones that are a bit harder so that we can get some discussion around them, since my Ruby is still quite nooob status

Reply:
I thought that might be the case (not the noob bit), good thinking :D